spec/index.bs

```diff
@@ -15736,8 +15736,9 @@ This stage produces a <dfn dfn>Fragment</dfn> for each [=RasterizationPoint=]:
         |state|.{{RenderState/[[stencilReference]]}}, "[=aspect/stencil=]", |state|.{{RenderState/[[depthStencilAttachment]]}},
         |stencilState|?.{{GPUStencilFaceState/compare}}).
     1. If |fragmentDesc| is not `null`:
-        1. If |fragment|.[=Fragment/depthPassed=] is `false` and the `frag_depth` [=builtin=] is not produced by the
-            shader the following steps may be skipped.
+        1. If |fragment|.[=Fragment/depthPassed=] is `false`, the `frag_depth` [=builtin=] is not produced by the
+            shader, and the shader does not write to any [=internal usage/storage=] bindings, the following steps may
+            be skipped.
```

Comment:
nit: `shader` -> `shader entry point` or something like that, in both places in this sentence

Reply:
Done!